rootfs/requirements.txt

```diff
@@ -10,7 +10,8 @@ gunicorn==19.4.1
 psycopg2==2.6.1
 python-etcd==0.3.2
 PyYAML==3.11
-requests==2.8.1
-simpleflock==0.0.2
+requests==2.9.1
+#simpleflock==0.0.2
+git+https://github.com/helgi/python-simpleflock.git@python3#egg=simpleflock
```

Comment:
waiting on upstream to merge, doubt they will tho - couldn't find any proper alternatives yet. Best idea is to remove `simpleflock` and related code if at all feasible